rotkehlchen/db/upgrade_manager.py

```diff
@@ -212,46 +211,38 @@ def _perform_single_upgrade(
         progress_handler.new_round(version=to_version)
 
         # First make a backup of the DB
-        with TemporaryDirectory() as tmpdirname:
-            tmp_db_filename = f'{ts_now()}_rotkehlchen_db_v{upgrade.from_version}.backup'
-            tmp_db_path = os.path.join(tmpdirname, tmp_db_filename)
-            shutil.copyfile(
-                os.path.join(self.db.user_data_dir, USERDB_NAME),
-                tmp_db_path,
-            )
-
-            # Add a flag to the db that an upgrade is happening
-            with self.db.user_write() as write_cursor:
-                self.db.set_setting(
-                    write_cursor=write_cursor,
-                    name='ongoing_upgrade_from_version',
-                    value=upgrade.from_version,
-                )
+        tmp_db_filename = f'{ts_now()}_rotkehlchen_db_v{upgrade.from_version}.backup'
+        shutil.copyfile(
+            os.path.join(self.db.user_data_dir, USERDB_NAME),
+            os.path.join(self.db.user_data_dir, tmp_db_filename),
+        )
```

Comment:
one thing that I don't know if it happens before creating the backup... do we consume the wal file before copying the db?

Reply:
Good questions. Not sure. What can we do to make sure it is? 

We either need to do this or copy/backup all files. Whcih imo is senseless. Same question for the globalDB upgrade too. Since you dealt with the WAL can you find out?

Since this is at th every very start of the app I guess it's fine? But if there is multipel upgrades one after the other then we need to be more careful?

Reply:
done